cli/gh-pr-filters.go

```diff
@@ -89,18 +89,17 @@ func GetFilterForMilestone(milestoneRaw string) *Filter {
 		PR: func(pr github.PullRequest) (bool, error) {
 			milestone := pr.GetMilestone().GetTitle()
 
-			//nolint:gocritic
 			if strings.EqualFold(filterMilestone, milestone) && !negate {
 				c.Printf("    milestone: <green>%s</> <gray>(%s)</>\n", filterMilestone, milestone)
 				return true, nil
 			} else if negate {
 				c.Printf("    milestone: <green>-%s</> <gray>(%s)</>\n", filterMilestone, milestone)
 				return true, nil
-			} else {
-				//revive:disable:indent-error-flow
-				c.Printf("    milestone: <red>%s</> <gray>(%s)</>\n", filterMilestone, milestone)
-				return false, nil
 			}
+			//revive:disable:indent-error-flow
```

Comment:
why are you disabling this

Reply:
it was a mistake, sorry, i'll take it out